### ISSUE TITLE
[Bug]: Some mapped chunks are being forgotten

### ISSUE THREAD

**Issue by Salamance73** [FTBTeam/FTB-Mods-Issues#395]
### Mod

FTB Chunks

### Mod version

ftb-chunks-forge-1802.3.6-build.147

### Forge / Fabric version

1.18.2 40.1.60

### Modpack & version

All The Mods 7 0.4.23

### What issue are you having?

After mapping an area over several game regions, many of the chunks in some regions (unsure of criteria) are seemingly forgotten or not being loaded until I begin to remap that region. Below is a screenshot of my house area that should be completely round in the map (I have deleted all map data from the local folder), but along the region border the mapped chunks are not showing up.

map area pre-arrival:
![image](https://user-images.githubusercontent.com/26430914/182056618-3a406d76-9a2e-40d3-b743-df5744ae0484.png)

map area post-arrival:
![image](https://user-images.githubusercontent.com/26430914/182056888-734c0bd2-7a8c-4b01-9040-8e6e16a65bd7.png)

### Steps to reproduce

1. starting with an empty map folder, load a single area that crosses over a region border
2. teleport away from the area completely but keep track of where it is on the map
3. log out
4. log back in
5. before, teleporting back to the mapped area, allow the map to load and check the previously-mapped area 
6. observe missing chunks 

### Anything else to note?

I recently updated my modpack from v0.4.21 and attempted to transfer my map over, as seen now it was seemingly blank until I walked around in the previously mapped regions one by one.

This is also not a single player world.

**Comment by MichaelHillcox** [comment]
Yeah! Sorry about the... _checks notes_... 3 year wait... Glad you're still enjoying modded minecraft :heart:
